htmlfind/htmlfind.go

```diff
@@ -0,0 +1,67 @@
+package htmlfind
```

Comment:
Should we consider putting this in an `internal` subdirectory?

Reply:
Agree.